docs/developers/guides/gas/gas-fees.mdx

```diff
@@ -12,9 +12,19 @@ import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem';
 
 Linea supports the [Ethereum EIP-1559 gas price model](https://ethereum.org/developers/docs/gas). 
 However, as a layer 2 blockchain, Linea provides a more stable and cost-effective solution 
-for transaction fees.
+for transaction fees. The EIP-1559 model formula is:
+```
+units of gas used * (base fee + priority fee)
+```
+
+Using parameters in the [Linea source code](https://github.com/Consensys/linea-monorepo/blob/a001342170768a22988a29b2dca8601199c6e205/sdk/src/clients/blockchain/gas/LineaGasProvider.ts), 
+the formula can be rendered as:
+```
+gasLimit * (maxBaseFeePerGas + maxPriorityFeePerGas)
+```
 
-There are minor differences in the way Linea handles gas calculations when compared with Ethereum:
+However, there are minor differences in the way Linea handles gas calculations when compared with 
+Ethereum:
 
 - **The base fee uses a set price of 7 wei.** Blocks created by Linea use up to 24 million gas
     (less than 50% of the maximum Linea block size of 61 million gas), and the fee decreases by 
```

Comment:
Transactions won't be sequenced if the gasPrice or maxPriorityFeePerGas falls below a certain threshold

This section is quite confusing. The important part to highlight is that the rest of the max fee is the priority fee, and just like ethereum the max fee needs to be higher than the current gas price for inclusion